Stripe/STPPaymentCardTextField.m

```diff
@@ -161,7 +161,7 @@ - (void)commonInit {
     expirationField.alpha = 0;
     expirationField.accessibilityLabel = STPLocalizedString(@"expiration date", @"accessibility label for text field");
     self.expirationField = expirationField;
-    self.expirationPlaceholder = @"MM/YY";
+    self.expirationPlaceholder = STPLocalizedString(@"MM/YY", "label for text field to enter card expiry");
```

Comment:
Looks like CI failed because this is missing `@` at the beginning of the second string here.